projects/packages/stats/tests/php/test-options.php

```diff
@@ -0,0 +1,388 @@
+<?php // phpcs:ignore WordPress.Files.FileName.InvalidClassFileName
+/**
+ * Tests Options class.
+ *
+ * @package jetpack-stats
+ */
+
+namespace Automattic\Jetpack\Stats;
+
+use Automattic\Jetpack\Constants;
+use Jetpack_Options;
+use WorDBless\BaseTestCase;
+use WorDBless\Options as WorDBless_Options;
+
+/**
+ * Class to test the Options class.
+ *
+ * @covers Options
+ */
+class Test_Options extends BaseTestCase {
+	/**
+	 * Set up before each test
+	 *
+	 * @before
+	 */
+	protected function set_up() {
+		parent::setUp();
+		Constants::set_constant( 'STATS_VERSION', '9' );
```

Comment:
This value is used in many places. Maybe extract it in a constant?

Reply:
I need to add lots of unit tests in this package while adding the corresponding functionalities. I might need to introduce a base test class for setting up / tearing down all tests so I'll move that there instead in a future PR. Hope that's ok :)